x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/date_histogram.tsx

```diff
@@ -334,3 +347,138 @@ function restrictedInterval(aggregationRestrictions?: Partial<IndexPatternAggRes
     aggregationRestrictions.date_histogram.fixed_interval
   );
 }
+
+const AutoDateHistogramPopover = ({ data }: { data: DataPublicPluginStart }) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const autoDateHistogramHelpPopoverTitle = i18n.translate(
+    'xpack.lens.indexPattern.dateHistogram.titleHelp',
+    {
+      defaultMessage: 'How does the auto date histogram work?',
+    }
+  );
+  const infiniteBound = i18n.translate('xpack.lens.indexPattern.dateHistogram.moreThanYear', {
+    defaultMessage: 'More than a year',
+  });
+  const upToLabel = i18n.translate('xpack.lens.indexPattern.dateHistogram.upTo', {
+    defaultMessage: 'Up to',
+  });
+
+  const formatterOptions = {
+    inputFormat: 'milliseconds',
+  };
+
+  const humanDurationFormatter = data.fieldFormats.deserialize({
+    id: 'duration',
+    params: formatterOptions,
+  });
+
+  return (
+    <EuiPopover
+      ownFocus
+      isOpen={isPopoverOpen}
+      button={
+        <EuiLink onClick={() => setIsPopoverOpen(!isPopoverOpen)}>
```

Comment:
It wasn't obvious that this was clickable to me. How about making it `primary` color or adding an icon?